driver-sync/src/test/functional/com/mongodb/client/AbstractRetryableReadsTest.java

```diff
@@ -137,6 +138,12 @@ public void apply(final SocketSettings.Builder builder) {
                         builder.readTimeout(5, TimeUnit.SECONDS);
                     }
                 })
+                .applyToServerSettings(new Block<ServerSettings.Builder>() {
```

Comment:
Shane suggested this to make these tests run faster.

Reply:
Ack